### ISSUE TITLE
In stream subtitles DASH not working with http-streaming project 

### ISSUE THREAD

**Issue by Fruitsapje** [videojs/http-streaming#391]
## Description
Hi guys,

When moving over from `contrib-dash` to the `http-streaming` project I have encountered an issue where the in stream subtitles for a DASH stream are no longer working. The console is getting the following warning: `VIDEOJS: WARN: Error encountered when parsing cues: Malformed WebVTT signature.`

The test stream is showing this error when testing on the `http-streaming` test page:
https://videojs.github.io/http-streaming/
But it is working correctly on the `contrib-dash` test page:
https://videojs.github.io/videojs-contrib-dash/

## Sources
Test `.mpd` with in stream subtitles (don't mind the language label not matching the actual language):
https://s1-media.streamone.io/dash/account=MpBJPKUJ4Y6i/item=lAAspcbCJVCm/lAAspcbCJVCm.mpd

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.

1. Go to either the `contrib-dash` or `http-streaming` test page
2. Paste in the `.mpd` test URL
3. Press play and seek abit for subtitles to show

### Expected
Expected both test pages to display the in stream subtitles.

### Error output
in the `http-streaming` test page:
`VIDEOJS: WARN: Error encountered when parsing cues: Malformed WebVTT signature.`

## Additional Information
Please include any additional information necessary here. Including the following:

### videojs-http-streaming version
what version of videojs-http-streaming does this occur with?
videojs-http-streaming 1.4.1 and latest version on test page

### videojs version
what version of videojs does this occur with?
video.js 7.3.0 and latest version on test page

Hope you can help.

Thanks!

**Comment by hero24i** [comment]
Hi guys,

Any update available on this?

**Comment by Fruitsapje** [comment]
Hey there,

Is there any update available on this matter? Would love some feedback so I can make some decisions regarding this subject :)

Hope to hear from you!

Kind regards

**Comment by hrehman200** [comment]
I am also facing the same problem with videojs 7 with http-streaming built in. Subtitles show properly in Safari if I set nativeTextTracks: true

**Comment by AlexGruzin** [comment]
Same problem 
`"video.js": "7.13.1"`
My config:
```
{
        autoplay: 'any',
        responsive: true,
        controls: true,
        preload: 'auto',
        nativeAudioTracks: false,
        nativeVideoTracks: false,
        nativeTextTracks: false,
        nativeControlsForTouch: false,
        preloadTextTracks: true,
        html5: {
            nativeCaptions: false,
            vhs: {
                enableLowInitialPlaylist: true,
                handlePartialData: true,
                parse708captions: true,
                overrideNative: true,
                handleManifestRedirects: true,
            }
        }
    }
```
No errors I could see. Subs are shown in TextTrackList(), but enabling them makes no effect.
So as I see contrib-dash making requests to download text tracks (screenshoot). Vhs do not make text tracks requests.

<img width="888" alt="Screenshot 2021-06-16 at 18 23 20" src="https://user-images.githubusercontent.com/23658183/122247337-f0e4a300-cecf-11eb-98c6-64556a4cd6b0.png">

Here is the source to check:
url: 'https://storage.googleapis.com/shaka-demo-assets/sintel-mp4-wvtt/dash.mpd' (nothing works)
or
url: 'https://livesim.dashif.org/livesim/testpic_2s/cea608.mpd' (sw track not working)

**Comment by gkatsev** [comment]
Unfortunately, we currently only support VTT based captions in DASH, not those embedded in mp4.

**Comment by phloxic** [comment]
@gkatsev - the first example is arguably vtt based, but embedded in mp4; do you mean a 'normal' vtt file (not in mpd), or sidecar ttml (never tried that), or?

**Comment by gkatsev** [comment]
Yup, vtt embedded in mp4 isn't supported. Direct vtt files are. TTML isn't supported either right now, unfortunately.

**Comment by phloxic** [comment]
> Yup, vtt embedded in mp4 isn't supported. Direct vtt files are. TTML isn't supported either right now, unfortunately.

Thanks for clarifying.

**Comment by phloxic** [comment]
For completeness sake, what also seems to work are 'sidecar' vtt referenced in mpd, sth like:
```xml
  <AdaptationSet mimeType="text/vtt" lang="en">
    <Representation id="1" bandwidth="256">
      <BaseURL>https://mydomain.com/subtitles-en.vtt</BaseURL>
    </Representation>
  </AdaptationSet>
```
